package.json

```diff
@@ -15,26 +15,32 @@
     "karma": "karma start",
     "karma-ci": "karma start karma-ci.conf.js",
     "karma-ci-cov": "karma start karma-ci-cov.conf.js",
-    "test": "npm run lint && npm run karma-ci-cov"
+    "test": "npm run lint && npm run karma-ci-cov",
+    "wdi5": "wdio run ./webapp/test/e2e/wdio.conf.ts"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/SAP-samples/ui5-typescript-helloworld.git"
   },
   "devDependencies": {
     "@types/openui5": "1.116.1",
-    "@typescript-eslint/eslint-plugin": "^6.3.0",
-    "@typescript-eslint/parser": "^6.3.0",
-    "@ui5/cli": "^3.4.0",
-    "eslint": "^8.46.0",
+    "@typescript-eslint/eslint-plugin": "^6.7.0",
+    "@typescript-eslint/parser": "^6.7.0",
+    "@ui5/cli": "^3.6.0",
+    "@wdio/cli": "^8",
+    "@wdio/local-runner": "^8",
+    "@wdio/mocha-framework": "^8",
+    "@wdio/spec-reporter": "^8",
+    "eslint": "^8.49.0",
     "karma": "^6.4.2",
     "karma-chrome-launcher": "^3.2.0",
     "karma-coverage": "^2.2.1",
     "karma-ui5": "^3.0.3",
-    "karma-ui5-transpile": "^3.0.1",
+    "karma-ui5-transpile": "^3.2.3",
     "rimraf": "^5.0.1",
-    "typescript": "^5.1.6",
+    "ts-node": "^10.9.1",
```

Comment:
Care to elaborate on this change? "tsc" is used above, will it still work with ts-node? It's true that typescript is not used otherwise, but then again ts-node also does not appear to be used directly.

Reply:
`wdi5`/`wdio` uses `ts-node` for on-the-fly transpiling TS-files to JS - including the tests themselves.
dropping `typescript` wasn't intended. 
will double-check together with the few minor corrections mentioned above.